packages/ui/src/utils/ThemeUtil.ts

```diff
@@ -1,231 +1,433 @@
-      Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
-
-    --wui-font-size-micro: 10px;
-    --wui-font-size-tiny: 12px;
-    --wui-font-size-small: 14px;
-    --wui-font-size-paragraph: 16px;
-    --wui-font-size-large: 20px;
-
-    --wui-font-weight-regular: 500;
-    --wui-font-weight-medium: 600;
-    --wui-font-weight-bold: 700;
-
-    --wui-letter-spacing-large: -0.8px;
-    --wui-letter-spacing-paragraph: -0.64px;
-    --wui-letter-spacing-small: -0.56px;
-    --wui-letter-spacing-tiny: -0.48px;
-    --wui-letter-spacing-micro: -0.2px;
-
-    --wui-color-inherit: inherit;
-
-    --wui-border-radius-5xs: 4px;
-    --wui-border-radius-4xs: 6px;
-    --wui-border-radius-3xs: 8px;
-    --wui-border-radius-xxs: 12px;
-    --wui-border-radius-xs: 16px;
-    --wui-border-radius-s: 20px;
-    --wui-border-radius-m: 28px;
-    --wui-border-radius-l: 36px;
-    --wui-border-radius-3xl: 80px;
-
-    --wui-icon-box-size-xs: 20px;
-    --wui-icon-box-size-sm: 24px;
-    --wui-icon-box-size-md: 32px;
-    --wui-icon-box-size-lg: 40px;
-
-    --wui-icon-size-inherit: inherit;
-    --wui-icon-size-xxs: 10px;
-    --wui-icon-size-xs: 12px;
-    --wui-icon-size-sm: 14px;
-    --wui-icon-size-md: 16px;
-    --wui-icon-size-lg: 20px;
-
-    --wui-wallet-image-size-inherit: inherit;
-    --wui-wallet-image-size-sm: 40px;
-    --wui-wallet-image-size-md: 56px;
-    --wui-wallet-image-size-lg: 80px;
-
-    --wui-spacing-0: 0px;
-    --wui-spacing-4xs: 2px;
-    --wui-spacing-3xs: 4px;
-    --wui-spacing-xxs: 6px;
-    --wui-spacing-2xs: 7px;
-    --wui-spacing-xs: 8px;
-    --wui-spacing-1xs: 10px;
-    --wui-spacing-s: 12px;
-    --wui-spacing-m: 14px;
-    --wui-spacing-l: 16px;
-    --wui-spacing-2l: 18px;
-    --wui-spacing-xl: 20px;
-    --wui-spacing-xxl: 24px;
-    --wui-spacing-3xl: 40px;
-
-    --wui-ease-out-power-2: cubic-bezier(0, 0, 0.22, 1);
-    --wui-ease-out-power-1: cubic-bezier(0, 0, 0.55, 1);
-
-    --wui-ease-in-power-3: cubic-bezier(0.66, 0, 1, 1);
-    --wui-ease-in-power-2: cubic-bezier(0.45, 0, 1, 1);
-    --wui-ease-in-power-1: cubic-bezier(0.3, 0, 1, 1);
-
-    --wui-ease-inout-power-1: cubic-bezier(0.45, 0, 0.55, 1);
-
-    --wui-duration-lg: 200ms;
-    --wui-duration-md: 125ms;
-    --wui-duration-sm: 75ms;
-
-    --wui-cover: rgba(0, 0, 0, 0.3);
-
-    --wui-box-shadow-blue: rgba(51, 150, 255, 0.16);
-
-    --wui-z-index: 89;
-
-    --wui-box-size-md: 100px;
-    --wui-box-size-lg: 120px;
-
-    --wui-path-network: path(
-      'M43.4605 10.7248L28.0485 1.61089C25.5438 0.129705 22.4562 0.129705 19.9515 1.61088L4.53951 10.7248C2.03626 12.2051 0.5 14.9365 0.5 17.886V36.1139C0.5 39.0635 2.03626 41.7949 4.53951 43.2752L19.9515 52.3891C22.4562 53.8703 25.5438 53.8703 28.0485 52.3891L43.4605 43.2752C45.9637 41.7949 47.5 39.0635 47.5 36.114V17.8861C47.5 14.9365 45.9637 12.2051 43.4605 10.7248Z'
-    );
+export function setThemeVariables(themeVariables: ThemeVariables) {
+  if (typeof document !== 'undefined') {
```

Comment:
Not sure if this is the best way to do this i.e. if we have modal open for some sort of "real time" editing, we probably just want to override data, not destroy and re-create it. Perhaps simple

```
styleTag.textContent = createRootStyles(themeVariables).cssText
```

here will do the trick on already existing styleTag?